README.md

```diff
@@ -1 +1,18 @@
-# extract-id3-tags-from-mp3s
+# Extract ID3 Tags From MP3s
+This program extracts id3 tags from an mp3 file, allowing automated metadata collection
```

Comment:
Follow standard-readme by adding the description set in the package.json in a `>` quote.